source/_integrations/devolo_home_network.markdown

```diff
@@ -34,6 +34,13 @@ The **devolo Home Network** {% term integration %} integration allows you to mon
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+IP address:
+  description: "IP address of your devolo Home Network device. This can be found in the devolo Home Network App on the device dashboard."
+  required: true
+  type: string
```

Comment:
```suggestion
```
Not valid to use inside `configuration_basic`

Reply:
hmm, it seemed I copied from a bad example, but I cannot tell which one anymore. I removed it 👍